sdcm/argus_results.py

```diff
@@ -126,16 +126,34 @@ class Meta:
         name = "Backup benchmark"
         description = "Backup benchmark"
         Columns = [
-            ColumnMetadata(name="backup time", unit="s", type=ResultType.DURATION, higher_is_better=False),
+            ColumnMetadata(name="Size", unit="bytes", type=ResultType.TEXT, higher_is_better=False),
+            ColumnMetadata(name="Time", unit="s", type=ResultType.DURATION, higher_is_better=False),
         ]
 
 
-class ManagerBackupReadResult(GenericResultTable):
+class ReadStressResult(GenericResultTable):
     class Meta:
         name = "Read timing"
         description = "Read timing"
         Columns = [
             ColumnMetadata(name="read time", unit="s", type=ResultType.DURATION, higher_is_better=False),
+            ColumnMetadata(name="op rate", unit="op/s", type=ResultType.FLOAT, higher_is_better=True),
+            ColumnMetadata(name="partition rate", unit="pk/s", type=ResultType.FLOAT, higher_is_better=True),
+            ColumnMetadata(name="row rate", unit="row/s", type=ResultType.FLOAT, higher_is_better=True),
+            ColumnMetadata(name="latency 99th percentile", unit="ms", type=ResultType.FLOAT, higher_is_better=False),
+        ]
+
+
+class WriteStressResult(GenericResultTable):
```

Comment:
Dont we have something similar in performance tests alraedy?

Reply:
replaced and removed in new second commit